src/System.Configuration.ConfigurationManager/src/Resources/Strings.resx

```diff
@@ -94,10 +94,7 @@
   </data>
   <!--
     other exceptions
-  -->
-  <data name="Argument_AddingDuplicate" xml:space="preserve">
```

Comment:
BTW I see an ID elsewhere "Argument_AddingDuplicate__" that could just be "Argument_AddingDuplicate"

Reply:
done